Gaomon_M106K/index.md

```diff
@@ -0,0 +1,19 @@
+---
+VID: "256c"
+PID: "006e"
+vendor: Huion
+product: Gaomon
```

Comment:
Oh and these should be `Gaomon` and `M106K`. Fixing now.

Reply:
Ah, wait, I think this is simply a rebranding of Huion New 1060 Plus. Oh well, will try to figure it out somehow.